shell/platform/darwin/ios/framework/Source/FlutterPlatformViews.mm

```diff
@@ -731,6 +732,7 @@ - (instancetype)initWithTarget:(id)target
 @implementation FlutterTouchInterceptingView {
   fml::scoped_nsobject<DelayingGestureRecognizer> _delayingRecognizer;
   FlutterPlatformViewGestureRecognizersBlockingPolicy _blockingPolicy;
+  UIView* _embeddedView;
```

Comment:
Since we have the DCheck, I don't think we need this any more. Because plugin developers will get the crash when they develop on their plugin and fix the bug.

Reply:
@cyanglaz 👌 Done.